src/lib.rs

```diff
@@ -92,61 +99,99 @@ where
+                let mut span = iter.next().expect(
+                    "Unable to get the next item in the iterator; this should not be possible.",
+                );
+                loop {
+                    let ext = span.extensions();
+                    let fields = &ext
+                        .get::<FormattedFields<N>>()
+                        .expect("will never be `None`");
+
+                    let fields = if !fields.is_empty() {
+                        Some(fields.as_str())
+                    } else {
+                        None
+                    };
+
+                    let fields = FormatSpanFields::format_fields(
+                        span.name(),
+                        fields,
+                        writer.has_ansi_escapes(),
+                    );
+                    write!(writer, "{}", fields)?;
+
+                    drop(ext);
+                    match iter.next() {
+                        // if there's more, add a space.
+                        Some(next) => {
+                            write!(writer, ", ")?;
+                            span = next;
+                        }
+                        // if there's nothing there, close.
+                        None => break,
                     }
-                    // if there's nothing there, close.
-                    None => break,
                 }
+                write!(writer, "] ")?;
             }
-            write!(writer, "] ")?;
         }
-
         ctx.field_format().format_fields(writer.by_ref(), event)?;
         writeln!(writer)
     }
 }
 
+impl Glog {
+    /// Sets whether or not the span context is included. Defaults to true.
+    ///
+    /// By default, formatters building atop of [`mod@tracing_subscriber::fmt`]
+    /// will include the span context as [`fmt::Layer`] and
+    /// [`fmt::Subscriber`] assume that span context is
+    /// is valuable and the _raison d’être_ for using `tracing` and, as such, do not provide a
+    /// toggle. However, users migrating from logging systems such
+    /// as [glog](https://github.com/google/glog) or folly's [`xlog`](https://github.com/facebook/folly/blob/main/folly/logging/xlog.h)
+    /// might find the span context to be overwhelming. Therefore, this formatter-level toggle
+    /// is availible in order to provide a smoother onboarding experience to [`tracing`].
+    ///
+    /// **Notice:** This is a relatively coarse toggle. In most circumstances, usage of
+    /// `tracing-subscriber`'s [`filter_fn`] is preferred to disable spans on a more
+    /// fine-grained basis.
+    ///
+    /// [`fmt::Layer`]: tracing_subscriber::fmt::Layer
+    /// [`fmt::Subscriber`]: tracing_subscriber::fmt::Subscriber
+    /// [per-layer filtering]: https://docs.rs/tracing-subscriber/latest/tracing_subscriber/layer/index.html#per-layer-filtering
+    /// [`filter_fn`]: fn@tracing_subscriber::filter::filter_fn
+    /// [`tracing`]: mod@tracing
```

Comment:
this should probably be a docs.rs link rather than a `mod` link, right? does `mod@` work with external crates?

Reply:
`mod@` does work for intradoc links if the linked crate is a dependency. I should probably switch most of the links over to intradoc link.

Reply:
huh, TIL!